solr/contrib/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrExporter.java

```diff
@@ -68,7 +66,7 @@
   private static final String[] ARG_CONFIG_FLAGS = {"-f", "--config-file"};
   private static final String ARG_CONFIG_METAVAR = "CONFIG";
   private static final String ARG_CONFIG_DEST = "configFile";
-  private static final String ARG_CONFIG_DEFAULT = "solr-exporter-config.xml";
+  private static final String ARG_CONFIG_DEFAULT = "conf/solr-exporter-config.xml";
```

Comment:
@HoustonPutman you removed the conf/ and I'm putting it back.  I'm not sure why it worked with it gone.  It may be related to other changes in this PR that it's needed again.  I tested that this works here both with `gw run` and via executing normally from the built assembly.  I didn't test Docker yet.

Reply:
So the reason I removed the `conf/` is because I was reading the file off of the classpath. That way you can run `solr-exporter` from anywhere, not just `contrib/prometheus-exporter`.

The reason it doesn't work anymore is that we aren't using the SolrResourceLoader in this PR. I have a fix made that should allow for both the default config to be read off of the classpath or other configs specified via a file location. I can push it to this branch if you don't mind.

Reply:
Sure; push if you want.  I don't think it's important to read off the classpath so long as it reads it from somewhere... but if it works, that's nice.